api/base/serializers.py

```diff
@@ -701,6 +701,25 @@ def is_valid(self, clean_html=True, **kwargs):
 
         return ret
 
+class JSONAPIRelationshipSerializer(ser.Serializer):
+    def relationship(self, obj):
```

Comment:
Thinking about this more, do you need this?  The relationship id can be found in `self.validated_data.get('_id')`.  If the original request had `{'data': null}`, then this would be `None`, and you could return None.  If there was some other problem before we got here, I would think you would have raised an exception.